libnmstate/nm/common.py

```diff
@@ -0,0 +1,32 @@
+#
```

Comment:
How about calling this file `gilib`? I am not a fan of generic names, lets try to make it focused.

BTW, this seems a syntactic refactoring which you could do in a separate commit, lowering the noise of this one big commit. It will also assure no logic changes with this change (unlike the plugin changes).

Reply:
Renamed.

Reply:
No. I will keep it as `common.py` as it hold both glib stuff and NM.

As we renamed `nmclient.py` to `mainloop.py` which clearly not a good place to hold these.

Reply:
I won't block this PR on this one, but I think the name should be improved.